twilight-model/src/channel/message/component/label.rs

```diff
@@ -0,0 +1,21 @@
+use serde::{Deserialize, Serialize};
+
+use super::Component;
+
+/// Top-level layout [`Component`].
+///
+/// Labels wrap modal components with text as a label and an optional description.
+#[derive(Clone, Debug, Eq, Hash, PartialEq, Deserialize, Serialize)]
```

Comment:
Some components are `Deserialize, Serialize` while others aren't. Is it correct that this component should implement these traits?